concepts/arrays/about.md

```diff
@@ -126,33 +135,23 @@ let mut a = ArrayTrait::new();
 }
```

Comment:
Instead of a comment stating `` // Now `first` will be 0 and `second` will be 1``, write 2 `println` statements printing the values